### DISCUSSION TITLE
0.11.6-rc Data API builder for Azure Databases

### DISCUSSION THREAD

**Discussion by seantleonard** [Azure/data-api-builder#2035] [Announcements]
## Changes:

* f557b4da4c350f878d9a44632037daef9349f769 GraphQL stored procedure support for DW (#2008)
* 302520d429c119a22a8e6517f3f403a094d31ae1 Enhanced claims processing for MS SQL Set session context (#2003) [ #1957, #2004 ]
* a64c277a6d79317afbc8247180891b9a5a7cccb6  Cache enablement check and dab-draft.schema.json changes (#2033) [ #2011, #2012 ]
* d763a83c815dd73259ec659acae917c3449293cb Fix query string contents in nextLink for REST (remove dupe $after query parameter) (#2032) [ #2006, #1895, #00001, #00002 ]
* 3697d07520b3f69849f281e784415d64e35f6c28 Added Cosmos Db nested Support for array type of attributes (#2009) [ #1628, #1624 ]
* a102e44877b66ca78f6601d78f47dd02481c3416 Support environment variable serialization for data source options (#1980) [ #1956 ]
*  5fe65f8c0000bc68f0052c77131c223afa2e821f Make relationship fields in referenced entities nullable (#1958) [ #1747 ]
* 1f3f176aac39f3d923d125caa9b7724f3f49ac31 Add support for app insights in CLI (#1952) [ #1745 ]
* 0217d3ecf38e62c5af6e209854793dfaa94c477b Verifying the read access of anonymous users prior to transmitting the Mutation result. (#1893) [ #1464 ]

<details><summary><b>See More</b></summary>

### DW
* 462eb00bdc34760d95e2e4a77512597d716c5672 fixing datetime and bit issue in DW. (#1931)
* 542ac929f59617057006df427c364d937f491417 GraphQL Mutation support for datawarehouse. (#1978)

### Fundamentals and Pipelines
* 1a72c9c3e253ee2435bd50953c6516aa67afd48c Add tests for built_in methods support as default field values (#1986) [ #1857, #2014 ]
* 02463a22b44104ba9d549a73c9e59815e2cba1ae Update README.md (#2028)
* 27b652c22ecb952f5c402348c6374a269fd18535 Updating Service Connection for publishing dab container images (#2005)
* 02eed81b066884cc22bf322e4a1de4605b0591bf Update build-pipelines.yml (#1996)
* e681aa5709250309ed1d1b212808c913e4245eef Migrate MYSQL and PGSQL Integration test pipeline (#1993) [ #1923 ]
* 4d53e7e8fe78497cd74f3487037dd2fe0bf31594 Exposing the field mappings helpers through interface. (#1965)
* 95549b1b0780507285b2155c3f33f20c87ce3744 Cleanup of redundant class (#1963)
* aa577bf8ff029a74c82503e8166d266aa9a8cd9f Update Dependencies and Suppress NU1603 (#1970)
* dacf3bccbe0a36da51776bf4afd1b4a0d4348ff4 Add design doc for dab validate (#1822)
* bc146f370cc0a98b363bb56f84e3cbb329f74ba2 Upgrade deprecated tasks in pipeline (#1954)
* 1d94cccaba9580aecf7988c98573576d0cea9274 Update dwsql-pipelines.yml (#1955)
* 2902ccbc9ccdf22ffe9b54bb68b2639d14fbcebc Helper methods to expose field mappings (#1918)
* 28205161c2cea38a237d81772cfb6b4c61aae678 update mssql pipeline for migration (#1928) [ #1923 ]

This list of changes was [auto generated](https://msdata.visualstudio.com/CosmosDB/_build/results?buildId=118700416&view=logs).</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/Azure/data-api-builder/releases/tag/v0.11.6-rc'>0.11.6-rc Data API builder for Azure Databases</a>.</em>

**Comment by jaykar2020** [comment]
https://github.com/Azure/data-api-builder/commit/3697d07520b3f69849f281e784415d64e35f6c28 Added Cosmos Db nested Support for array type of attributes (https://github.com/Azure/data-api-builder/pull/2009) [ https://github.com/Azure/data-api-builder/issues/1628, https://github.com/Azure/data-api-builder/issues/1624 ]

I am on 0.11.6-rc and am not able to verify this fix. Multi-level nested object query still either returns 

Bad Request if nested entity/child type is NOT defined in config
-or-
No results if nested entity/child type is defined in config

Any idea what could be wrong? Happy to provide more info if needed.
